client/state/notices/README.md

```diff
@@ -42,16 +42,28 @@ this.props.successNotice(
 * `errorNotice()`
 * `infoNotice()`
 * `warningNotice()`
+* `updateNotice()`
 
 ## Options
 
 The first argument is the text to be displayed on the notice. The second argument is an optional object and accepts some properties:
 
 *All these parameters are optional*
 
-* `id` (default generated `uniqueId()` ) ID for your notice
+* `id` (default generated `uniqueId()` ) ID for your notice. If a notice with the same ID already exists then the new one will replace it.
+* `icon` (default null - use default icon) The icon name.
+* `button: 'Label'`: (optional) Text label to display on action button.
+* `href: 'https://wordpress.com'`: (optional, requires `button` to be set as well) Url to be used for the button action.
+* `onClick: function() {}`: (optional, requires `button` to be set as well) Function to be invoked for the button action.
 * `duration` (default null - notices stay forever) Duration in milliseconds to display the notice before dismissing.
 * `showDismiss` (default true) To indicate if dismiss button should be rendered within the overlay.
 * `isPersistent` (default false - notices disappear when navigating route) - should notice be persistent between route changes?
 * `displayOnNextPage` (default false) - should notice appear on next route change?
+* `className` (default null) - Additional class names to be applied to the notice.
```

Comment:
No, we don't. I'll remove the option because it is not as useful as I intended at first.
The reason why I added the option was to show update notiice by extending `is-info` notice. 
Since it can be used to change color or icon of notice I kept the option to provide custom look even after adding `is-update`.